zxcvbn-cs/README.md

```diff
@@ -87,3 +87,14 @@ http://tech.dropbox.com/?p=165
 ### Licence
 
 Since `Zxcvbn-cs` is a port of the original `Zxcvbn` the original copyright and licensing applies. Cf. the LICENSE file.
+
+
+### Changes differ from mickford/zxcvbn-cs
+
+The following changes differ from the orginal one (mickford/zxcvbn-cs - https://github.com/mickford/zxcvbn-cs)
+
+- Add keyboard layouts: 
+	Swiss German, Swiss French, Italian, German, French
+		
+- Changed date matcher logics:
```

Comment:
Your writing stile switches between imperative, present simple and past simple. Choose one.

Reply:
Addressed in: https://github.com/messerli-informatik-ag/zxcvbn-cs/pull/1/commits/8f3cf4fda0d758466432e3b7f93d5982338047c9